### PR TITLE
Spanish Translation of README

### PR DESCRIPTION
**Pull request by L-David**
### Affected components

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by L-David** [comment]
This is a Spanish translation. I know there is a pull request for this in #3189, it seems that the other one has some issues. Also, this translation is in common Mexican Spanish form.

File: translations/README.spanish.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+* **Combina esquemas remotos**: Accede a esquemas personalizados de GraphQL para la lógica de negocio a través de un único punto de GraphQL Engine. [**Lee mas**](remote-schemas.md).
+* **Desencadena webhooks o funciones serverless**: En Postgres, insertar/actualizar/eliminar eventos ([Lee mas](event-triggers.md))
+* **Funciona con base de datos existentes y en uso**: Apúntalo a una base de datos Postgres existente para obtener instantáneamente una API GraphQL lista para usar.
+* **Control de acceso Fino**: Control de acceso dinámico que se integra con su sistema de autenticación (por ejemplo: auth0, firebase-auth).
+* **High-performance & low-footprint**: Imagen de Docker de ~15MB; ~50MB RAM @ 1000 req/s; trabaja en multinúcleo.
+* **UI de administración & Migraciones**: UI de administración & Esquemas de migraciones inspirados en rails.
+* **Postgres** ❤️: Soporte para los tipos de Postgres (PostGIS / geo-location, etc.), convierte las vistas en * gráficas *, activa funciones almacenadas o procedimientos con mutaciones
+
+Lee mas en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
```
Comment (Skao93) [inline translations/README.spanish.md]: Este «mas» lleva tilde.

File: translations/README.spanish.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine es un potente servidor GraphQL que provee de **APIs instantaneas y en tiempo real usando Postgres**, con [**disparadores de webhooks**](event-triggers.md) basados en los eventos de la base de datos, y [**esquemas remotos**](remote-schemas.md) para la lógica de negocio.
+
+Hasura te ayuda a construir aplicaciones GraphQL respaldadas por Postgres o permite a aplicaciones construidas con Postgres en moverte gradualmente a GraphQL.
+
+Lee mas en [hasura.io](https://hasura.io) y en [docs](https://docs.hasura.io).
```
Comment (Skao93) [inline translations/README.spanish.md]: Este «mas» lleva tilde.

File: translations/README.spanish.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+## Características
+
+* **Crea consultas poderosas**: Soprte para Filtrado, paginación, búsqueda de patrones, inserción masiva, actualización, eliminación de mutaciones.
+* **En tiempo real**: Convierta cualquier consulta GraphQL en una consulta de datos en tiempo real mediante suscripciones.
+* **Combina esquemas remotos**: Accede a esquemas personalizados de GraphQL para la lógica de negocio a través de un único punto de GraphQL Engine. [**Lee mas**](remote-schemas.md).
+* **Desencadena webhooks o funciones serverless**: En Postgres, insertar/actualizar/eliminar eventos ([Lee mas](event-triggers.md))
+* **Funciona con base de datos existentes y en uso**: Apúntalo a una base de datos Postgres existente para obtener instantáneamente una API GraphQL lista para usar.
+* **Control de acceso Fino**: Control de acceso dinámico que se integra con su sistema de autenticación (por ejemplo: auth0, firebase-auth).
+* **High-performance & low-footprint**: Imagen de Docker de ~15MB; ~50MB RAM @ 1000 req/s; trabaja en multinúcleo.
```
Comment (Skao93) [inline translations/README.spanish.md]: Este encabezado está sin traducir.

File: translations/README.spanish.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+![Hasura GraphQL Engine en tiempo real Demo](../assets/realtime.gif)
+
+-------------------
+
+## Características
+
+* **Crea consultas poderosas**: Soprte para Filtrado, paginación, búsqueda de patrones, inserción masiva, actualización, eliminación de mutaciones.
+* **En tiempo real**: Convierta cualquier consulta GraphQL en una consulta de datos en tiempo real mediante suscripciones.
+* **Combina esquemas remotos**: Accede a esquemas personalizados de GraphQL para la lógica de negocio a través de un único punto de GraphQL Engine. [**Lee mas**](remote-schemas.md).
```
Comment (Skao93) [inline translations/README.spanish.md]: Este «mas» lleva tilde.

File: translations/README.spanish.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Haga clic en el siguiente botón para implementar GraphQL Engine en Heroku con el complemento gratuito Postgres:
+
+    [![Implementación en Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+2. Abre la consola de Hasura
+
+   Visita `https://<app-name>.herokuapp.com` (*reemplaza \<app-name\> con el nombre de tu aplicación*) para abrir la consola de administración.
+
+3. Haga su primera consulta GraphQL
```
Comment (Skao93) [inline translations/README.spanish.md]: Si el texto está en segunda persona informal, hay que mantenerlo así. Acá iría «Haz tu primera consulta con GraphQL»

**Comment by marionschleifer** [comment]
@L-David could you also please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Pull request closed without merging**